harmony-api/src/services/userService.ts

```diff
@@ -2,6 +2,18 @@ import * as userRepo from '../repos/userRepo'
 import { User } from '../models/user'
 import { HttpError } from '../models/error/httpError'
 
+export const get = async (
+  userName?: string
+): Promise<User> => {
+  if (!userName) {
+    //return null
+  }
```

Comment:
This code isn't doing anything right now, should it maybe throw an error? something like
```suggestion
  if (!userName) {
    throw new HttpError('field username is required', 400)
  }
```

Reply:
I've been looking into how to redirect the user when an error occurs but haven't had any success, if you have any ideas I'd really appreciate them.

Reply:
Hmm, first thing that comes to mind is using the error field that is returned from `useHttpRequest` in a `useEffect`, and when a certain error is returned, redirect the user on the frontend. Just an idea